index.html

```diff
@@ -4,17 +4,289 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- Google Fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital@0;1&family=Quicksand:wght@300&display=swap" rel="stylesheet">   
     <!-- Here we import the Boostrap CSS stylesheet -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
-    <!-- Add a link to your custom CSS here -->
+    
+    <!-- Font Awesome -->
+    <script src="https://kit.fontawesome.com/846a6339d9.js" crossorigin="anonymous"></script>
+    <!-- CSS -->
+    <link rel="stylesheet" href="./css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <header class="p-0">
+        <nav class="display-md-col nav-padding navbar navbar-expand-md">
+            <!-- APPOINTMENT BTN SM DISPLAY -->
+            <div class="display-sm margin-bottom col-10">
+                <button class="appointment-btn">Book an Appointment</button>
+            </div>
 
-    <!-- Add your markup here -->
+            <button class="navbar-toggler p-0 margin-bottom" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <i class="col-1 navbar-toggler-icon fas fa-bars"></i>
+            </button>
+            <div class="display-sm row">
```

Comment:
Remember to use a `container` or `container-fluid` class before using `row`

Reply:
will do, think there may of been an issue why I didn't, but with it being common practice I should have done it and fixed the issue rather than leave it out.